src/libs/actions/Policy.js

```diff
@@ -1130,7 +1133,8 @@ function createDraftInitialWorkspace(policyOwnerEmail = '', policyName = '', pol
 function createWorkspace(policyOwnerEmail = '', makeMeAdmin = false, policyName = '', policyID = generatePolicyID()) {
     const workspaceName = policyName || generateDefaultWorkspaceName(policyOwnerEmail);
 
-    const {customUnits, customUnitID, customUnitRateID} = buildOptimisticCustomUnits();
+    const outputCurrency = lodashGet(allPersonalDetails, [sessionAccountID, 'localCurrencyCode'], CONST.CURRENCY.USD);
```

Comment:
We use the same line twice 
```js
const outputCurrency = lodashGet(allPersonalDetails, [sessionAccountID, 'localCurrencyCode'], CONST.CURRENCY.USD);
```

Can we move it to `buildOptimisticCustomUnits`? And for both cases (`createWorkspace` and `createDraftInitialWorkspace`) access it from `customUnits[customUnitID].rates[customUnitRateID].currency`